tools/releases/dockerfiles/kumactl.Dockerfile

```diff
@@ -5,3 +5,5 @@ ARG ARCH
 # override NOTICE
 COPY /tools/releases/templates/NOTICE-kumactl /kuma/NOTICE
 COPY /build/artifacts-linux-$ARCH/kumactl/kumactl /usr/bin/
+
+ENTRYPOINT ["/busybox/busybox", "sh", "-c"]
```

Comment:
Should we add this to `kumahq/base*`? I imagine we would have the same problem with `kumacp`. Also could you add tests for this? or at least have an issue to add tests for this?

Reply:
ok, all others have entrypoints

Reply:
and tests should appear with https://github.com/kumahq/kuma/pull/6581